Eevee.Sleep.Bot/Utils/DiscordMessageMaker/DiscordMessageMakerForRoleRestrict.cs

```diff
@@ -0,0 +1,69 @@
+using Discord;
+using Eevee.Sleep.Bot.Enums;
+using Eevee.Sleep.Bot.Extensions;
+
+namespace Eevee.Sleep.Bot.Utils.DiscordMessageMaker;
+
+public static class DiscordMessageMakerForRoleRestrict {
+    public static Embed MakeRestrictedRoleRemovedMessage(
+        string roleName,
+        IUser user,
+        uint accountAgeDays
+    ) {
+        var serverlink = $"https://discord.com/channels/{ConfigHelper.GetDiscordWorkingGuild()}";
```

Comment:
See if there's anyway to get the server name and change this to markdown style of link i.e. something like `[Pokemon Eevee Sleep](https://discord.com/channels/1138701819464392744)` instead of a plain link.